src/delete.test.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable import/no-duplicates */
+/* eslint-disable no-unused-vars */
+
+import onClickClear from './delete';
+import onClickDelete from './deleteOne';
+
+class LocalStorageMock {
+  constructor() {
+    this.store = {};
+  }
+
+  clear() {
+    this.store = {};
+  }
+
+  getItem(key) {
+    return this.store[key] || null;
+  }
+
+  setItem(key, value) {
+    this.store[key] = String(value);
+  }
+
+  removeItem(key) {
+    delete this.store[key];
+  }
+}
+
+document.body.innerHTML = `<input id="taskInput" value="Something">
```

Comment:
Kindly fix the error for the delete functionality test

![image](https://user-images.githubusercontent.com/34798898/127526937-7bf2d322-0c3d-4059-8139-98cd3f1ebfe6.png)